js/TxUtils.js

```diff
@@ -0,0 +1,57 @@
+function destinationsToAmount(destinations){
+    // Gets amount from destinations line
+    // input: "20.000000000000: 9tLGyK277MnYrDc7Vzi6TB1pJvstFoviziFwsqQNFbwA9rvg5RxYVYjEezFKDjvDHgAzTELJhJHVx6JAaWZKeVqSUZkXeKk"
+    // returns: 20.000000000000
+    return destinations.split(" ")[0].split(":")[0];
+}
+
+function destinationsToAddress(destinations){
+    var address = destinations.split(" ")[1];
+    if(address === undefined) return ""
+    return address;
+}
+
+function addressTruncate(address){
+    return address.substring(0, 6) + "..." + address.substring(address.length-6);
+}
+
+function check256(str, length) {
+    if (str.length != length)
+        return false;
+    for (var i = 0; i < length; ++i) {
+        if (str[i] >= '0' && str[i] <= '9')
+            continue;
+        if (str[i] >= 'a' && str[i] <= 'z')
+            continue;
+        if (str[i] >= 'A' && str[i] <= 'Z')
+            continue;
+        return false;
+    }
+    return true;
+}
+
+function checkAddress(address, testnet) {
+  return walletManager.addressValid(address, testnet)
+}
+
+function checkTxID(txid) {
+    return check256(txid, 64)
+}
+
+function checkSignature(signature) {
```

Comment:
@moneromooo-monero:  could you review this function?

Reply:
No review needed, was already in master; https://github.com/monero-project/monero-gui/blob/168a187f519727f82e17da65961ae2d0b974fc4b/pages/TxKey.qml#L66

There are 3 functions in `TxUtils.js` that are custom though: `destinationsToAmount()`, `destinationsToAddress()` and `addressTruncate()`.

Reply:
ok. thanks!